source/code/include/playfab/QoS/RegionResult.h

```diff
@@ -13,7 +13,7 @@ namespace PlayFab
         struct RegionResult
         {
         public:
-            RegionResult(std::string region, int latencyMs, int errorCode);
+            RegionResult(const std::string& region, int latencyMs, int errorCode);
```

Comment:
Doh man, another one. Yeah, clearly we overlooked a lot.

Reply:
It was an enum before, then hastily replaced to a string at some point. No wonder we missed it.. 

---
In reply to: [337638194](https://github.com/PlayFab/XPlatCoreTemplate/pull/24#discussion_r337638194) [](ancestors = 337638194)